extensions/cms/src/tasks/export.rs

```diff
@@ -49,9 +49,14 @@ impl TaskProvider for ExportTask {
         self.update(task).await?;
 
         println!("==> processing {} sboms for detail report", task.count);
+        let mut iteration = 0;
 
         for purl in primary_purls {
-            println!("==> generating detail report for purl {}", purl);
+            iteration += 1;
```

Comment:
Do you need this extra iterator?  You can use this variation of the for loop instead:

```
    for (i, el) in list.iter().enumerate() {
        println!("The current element is {}", el);
        println!("The current index is {}", i);
    }
    ```

Reply:
ahh that is an improvement.  Done.